docs/README.md

```diff
@@ -133,24 +126,22 @@ We've configured GitHub Actions to run the GitHub Pages workflow (gh-pages.yaml)
 
 ## Build the docs' static site files
 
-6. Run Sphinx with the `sphinx-multiversion` wrapper to build the doc's static site files. These files will be outputted to the `docs/build` directory.
+6. Run Sphinx with the `sphinx-multiversion` wrapper to build the doc's static site files. These files will be outputted to the `cleanlab-docs/` directory.
 
 ## Generate the `versioning.js` file used to store the latest release tag name and commit hash
```

Comment:
we should make it more clear this part is only relevant when releasing a new tagged version

Reply:
This part is run whenever the workflow is triggered by any of 3:
1. push to `master` branch
2. a new tag is released
3. manually run